app/models/patient.rb

```diff
@@ -69,6 +69,10 @@ def latest_phone_number
     phone_numbers.last&.number
   end
 
+  def latest_bp_passport
+    business_identifiers.where(identifier_type: "simple_bp_passport").order(:device_created_at).last
```

Comment:
This could be written as,

```suggestion
    business_identifiers.simple_bp_passport.order(:device_created_at).last
```

Reply:
Yeah... this scope isn't defined? Are you asking me to make it?

Reply:
Rails enums add scopes to models, since `identifier_type` is an enum, `simple_bp_passport` should be an available scope on `business_identifiers` already.